src/Makefile.am

```diff
@@ -377,6 +381,9 @@ bitcoind_LDADD += $(BOOST_LIBS) $(BDB_LIBS) $(SSL_LIBS) $(CRYPTO_LIBS) $(MINIUPN
 
 # bitcoin-cli binary #
 bitcoin_cli_SOURCES = bitcoin-cli.cpp
+if BUILD_EVUNIX
```

Comment:
@theuni This is pretty ugly, but I don't quite know how to do this otherwise. My first intuition was to add the file to `libbitcoin_util_a_SOURCES`, however these can't depend on libevent as that is shared between all executables. There is no library that is shared between just `bitcoind` and `bitcoin-cli` and adding one just for this seems overkill.